pallets/ocex/src/benchmarking.rs

```diff
@@ -273,29 +273,6 @@ benchmarks! {
 		}.into());
 	}
 
-	submit_snapshot {
-		<ExchangeState<T>>::put(true);
-		let snapshot = SnapshotSummary::decode(&mut SNAPSHOT.as_ref()).unwrap();
-		let call = Call::<T>::submit_snapshot { summary: snapshot };
-	}: { call.dispatch_bypass_filter(RawOrigin::None.into())? }
-	verify {
-		assert!(<Snapshots<T>>::contains_key(1));
-	}
```

Comment:
Extrinsic still [exists and tries to use weight](https://github.com/Polkadex-Substrate/Polkadex/blob/843169169b0400c279d380eefbbe0437955faa8d/pallets/ocex/src/lib.rs#L1015-L1020)  generated by this benchmark.

Reply:
Reverted